package.json

```diff
@@ -13,6 +13,9 @@
   },
   "dependencies": {
     "@apollo/client": "^3.5.8",
+    "@react-navigation/native": "^6.0.7",
+    "@react-navigation/native-stack": "^6.3.0",
```

Comment:
This dependency is not used as far as I can tell?

Reply:
Have you resolved this?

Reply:
Needed for the stack navigator